sdks/router-sdk/src/entities/mixedRoute/route.ts

```diff
@@ -4,6 +4,7 @@ import { Currency, Price, Token } from '@uniswap/sdk-core'
 import { Pair } from '@uniswap/v2-sdk'
 import { Pool as V3Pool } from '@uniswap/v3-sdk'
 import { Pool as V4Pool } from '@uniswap/v4-sdk'
+import { isValidTokenPath } from '../../utils/isValidTokenPath'
 
 type TPool = Pair | V3Pool | V4Pool
```

Comment:
nit: as I was coding v4 routing in SOR, I realized getting this type exported can be beneficial

Reply:
yah I also need to DRY and put this somewhere, I keep meaning to do taht

Reply:
I'm afraid of needing new approvals, so I'm just gonna merge..